assets/src/localStorage.ts

```diff
@@ -0,0 +1,34 @@
+import { RouteId } from "./skate"
+
+const APP_STATE_KEY = "mbta-skate-state"
+
+const SAVED_KEYS = ["selectedRouteIds"]
+
+interface PersistedState {
+  selectedRouteIds: RouteId[]
```

Comment:
Having `SAVED_KEYS` and `PersistedState` defined so far away from the full `State` makes me nervous that it will be easy for them to get out of sync. Perhaps, if this file becomes generic/reusable, then these could be defined in `state.ts`.